src/server.c

```diff
@@ -1570,6 +1588,20 @@ struct nvnc* nvnc_client_get_server(const struct nvnc_client* client)
 	return client->server;
 }
 
+EXPORT
+const char* nvnc_client_get_peer_hostname(const struct nvnc_client* client) {
```

Comment:
I wonder if maybe this should just be "nvnc_client_get_hostname".  I'm not sure if I ever properly understood the term "peer" in a network context, but it seems to suggest that two nodes are of equal footing. which they are not in this case.